documentation/modules/cruise-control/proc-cruise-control-moving-data.adoc

```diff
@@ -0,0 +1,339 @@
+If you are using JBOD storage and have Cruise Control installed with Strimzi, you can move reassign partitions and move data between the JBOD disks used for storage on the same broker.
+This capability also allows you to remove JBOD disks without data loss.
+
+To reassign partitions, configure a `KafkaRebalance` resource in `remove-disks` mode and specify a list of broker IDs with corresponding volume IDs for partition reassignment. 
+Cruise Control generates an optimization proposal based on the configuration and reassigns the partitions when approved manually or automatically. 
+
+Use the Kafka `kafka-log-dirs.sh` tool to check information about Kafka topic partitions and their location on brokers before and after moving them.
+Use an interactive pod to avoid running the tool within the broker container and causing any disruptions.
+
+.Prerequisites
+
+* xref:deploying-cluster-operator-str[The Cluster Operator must be deployed.]
+* xref:proc-configuring-deploying-cruise-control-str[Cruise Control is deployed with Kafka.]
+* Kafka operates in KRaft mode and brokers use JBOD storage.
+* More than one JBOD disk must be configured on the broker. 
+For more information on configuring Kafka storage, see xref:assembly-storage-str[].
+
+In this procedure, we use a Kafka cluster named `my-cluster`, which is deployed to the `my-project` namespace with node pools and Cruise Control enabled.
+
+.Example Kafka cluster configuration
+[source,yaml,subs="+attributes"]
+----
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+metadata:
+  name: my-cluster
+  namespace: my-project
+  annotations:
+    strimzi.io/node-pools: enabled
+spec:
+  kafka:
+    # ...
+  cruiseControl: {}
+    # ...
+----
+
+A node pool named `pool-a` is configured with three broker replicas that use three JBOD storage volumes.
+In the procedure, we show how partitions are reassigned from volume 1 and 2 to volume 0. 
+
+.Example node pool configuration with JBOD storage
+[source,yaml,subs=attributes+]
+----
+apiVersion: {KafkaNodePoolApiVersion}
+kind: KafkaNodePool
+metadata:
+  name: pool-a
+  labels:
+    strimzi.io/cluster: my-cluster
+spec:
+  replicas: 3
+  roles:
+    - broker
+  storage:
+    type: jbod
+    volumes:
+      - id: 0
+        type: persistent-claim
+        size: 2000Gi
+        deleteClaim: false
+      - id: 1
+        type: persistent-claim
+        size: 2000Gi
+        deleteClaim: false
+      - id: 2
+        type: persistent-claim
+        size: 2000Gi
+        deleteClaim: false
+  # ...
+----
+
+.Procedure
+
+. Run a new interactive pod container using the Kafka image to connect to a running Kafka broker.
++
+[source,shell,subs="+quotes,attributes"]
+----
+kubectl run --restart=Never --image={DockerKafkaImageCurrent} helper-pod -- /bin/sh -c "sleep 3600"
+----
++
+In this procedure, we use a pod named `helper-pod`.
+
+. Check the partition replica data on broker 0 by opening a terminal inside the interactive pod and running the Kafka `kafka-log-dirs.sh` tool:
```

Comment:
I think we should make clearer that this step is not mandatory. It can be used as a way to confirm (later) that the movement happened.